### DISCUSSION TITLE
WebStorm: NERDTree + Angular Schematic

### DISCUSSION THREAD

**Discussion by enginexon** [JetBrains/ideavim#724] [Q&A] [answered]
Hi!

I would like to call angular schematic from NERDTree by some custom key, for example `a`.
I tracked action id. It's `angularCliGenerate`
Is it posible?

Thank you.

**Comment by citizenmatt** [comment, marked as answer]
Unfortunately, it's not currently possible to create a custom mapping for the NERDTree extension. I've created an issue that would introduce a `NERDTreeAddKeyMap` function that should allow adding extra functionality. See [VIM-3121](https://youtrack.jetbrains.com/issue/VIM-3121/Support-NERDTreeAddKeyMap)

**Reply by enginexon** [reply]
Thank you!
